src/patterns/contact-a-department-or-service-team/index.md

```diff
@@ -9,9 +9,34 @@ layout: layout-pane.njk
 ---
 
 {%- from "_example.njk" import example -%}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
 
 Give users contact information within your service.
 
+{% set wcagCallout %}
+
+{{ govukTag({
+  text: "WCAG 2.2",
+  classes: "app-tag"
+}) }}
+
+### New WCAG 2.2 criteria affects this pattern
+
+To help users to 'Contact a department or service team' and meet the new WCAG 2.2 criteria, make sure that users can successfully:
+
+- [find contact details to get help in a consistent place on each page](/patterns/contact-a-department-or-service-team/#wcag-consistent-contact)
```

Comment:
```suggestion
- [find help details in a consistent place across contact pages](/patterns/contact-a-department-or-service-team/#wcag-consistent-contact)
```

Reply:
Same here, this doesn't clarify that it's for help with the same content. Although these links don't need to be that detailed, so don't necessarily need to change.